RELEASE.md

```diff
@@ -0,0 +1,53 @@
+Release
+=======
+
+Although not very tricky, the Ember Data release process does have a 
+lot of manual steps. The following steps navigate a through
+some of the release gotchas and will hopefully result in a successful
+release.
+
+STEPS:
+------
+
+* generate changelog (`PRIOR_VERSION=v2.0.0 HEAD=release ./bin/changelog`)
+* prepend changelog output to `CHANGELOG.md`
+* edit changelog output to be as user-friendly as possible (drop [INTERNAL] changes, non-code changes, etc.)
+* Bump version in package.json
```

Comment:
Don't know how obvious, but should there be a `git commit --all --message "Release Ember Data X.Y.Z"` after the bump?